src/components/TodoForm/TodoForm.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import { Form, Button, Alert } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import users from '../../api/users';
+
+export class TodoForm extends React.Component {
+  state = {
+    newTodo: {
+      todoTitle: '',
+      userName: '',
+    },
+    errors: {
+      todoError: false,
+      userError: false,
+    },
+  }
+
+  changeHandler = (e) => {
+    const { name, value } = e.target;
+
+    this.setState(prevState => ({
+      newTodo: {
+        ...prevState.newTodo,
+        [name]: value.trim(),
+      },
+    }));
+  }
+
+  submitHandler = (e) => {
```

Comment:
```suggestion
  handleSubmit = (e) => {
```

it's a function, use verbs